examples/networkpolicy/kubernetes-calico.json

```diff
@@ -3,26 +3,28 @@
   "properties": {
     "orchestratorProfile": {
       "orchestratorType": "Kubernetes",
+      "orchestratorRelease": "1.6",
+      "orchestratorVersion": "1.6.9",
```

Comment:
`orchestratorRelease` is a read-only property from a user input perspective. `acs-engine` template generation will choose the latest supported release version from the provided `orchestratorRelease` "major.minor" specification. So, we want to `orchestratorVersion` property from the examples api model here.

As an aside, should we increase the value of `orchestratorRelease` here to `"1.7"`, to reflect that the default kubernetes version is now on the 1.7 track?

Reply:
Ok, happy to only list 1.6 orchestratorRelease version and let the minor version be auto-selected.  There are problems with this version of calico and 2.5.1 as called out in the readme.   Calico 2.6 is in flight now and at that time it will be good to to K8s 1.7.x since there are fixes/improvements with integrations in 1.7.

Reply:
Got it.

We should add that check to validation here:

https://github.com/Azure/acs-engine/blob/master/pkg/api/vlabs/validate.go#L516

Something like:

```if networkPolicy == "calico" && a.OrchestratorProfile.OrchestratorRelease != "1.6" {
    return fmt.Errorf("<error string>")
}```

@lachie83 Are you okay with restricting Calico functionality to 1.6 clusters until calico 2.6 is baked?

Reply:
@jackfrancis I can add the validation you mention - makes sense until we get 2.6 calico assuming @lachie83 is good with this being set.

Could someone point me to the e2e validation test(s) for calico so I can test and potentially update that too?